src/coreclr/tools/Common/JitInterface/CorInfoBase.cs

```diff
@@ -2549,10 +2549,25 @@ static uint _getJitFlags(IntPtr thisHandle, IntPtr* ppException, CORJIT_FLAGS* f
             }
         }
 
+        [UnmanagedCallersOnly]
+        static byte _doesFieldBelongToClass(IntPtr thisHandle, IntPtr* ppException, CORINFO_FIELD_STRUCT_* fldHnd, CORINFO_CLASS_STRUCT_* cls)
+        {
+            var _this = GetThis(thisHandle);
+            try
+            {
+                return _this.doesFieldBelongToClass(fldHnd, cls) ? (byte)1 : (byte)0;
+            }
+            catch (Exception ex)
+            {
+                *ppException = _this.AllocException(ex);
+                return default;
```

Comment:
Shouldn't this be `(byte)0` as above?
I'm not sure why you would want to use `default` here

Reply:
I think the current version is correct. It is what most other calls are doing in case of exception, probably the idea is that the value won't be read when `ppException` is set.